apps/src/p5lab/spritelab/commands/criterionCommands.js

```diff
@@ -36,6 +36,29 @@ export const commands = {
     return result;
   },
 
+  // Return true if the specified sprite began speaking
+  // and the text was not an empty string.
```

Comment:
super nit: replace "was" with "is" in the new comments in this file. i think it helps clarify at a glance that we only care about the current frame, not anything that previously happened in the program